packages/components/tests/dummy/app/templates/utilities/dismiss-button.hbs

```diff
@@ -0,0 +1,52 @@
+{{page-title "DismissButton Component"}}
+
+<h2 class="dummy-h2">DismissButton</h2>
+
+<section>
+  <div class="dummy-banner dummy-banner--info">
+    <p class="dummy-paragraph">⚠️
+      <strong>Notice</strong>: this component is intended
+      <strong>only</strong>
+      for internal use (for now). If you need to use it please speak with the HDS team.</p>
+  </div>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <p class="dummy-paragraph" id="component-api-dismiss-button">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-dismiss-button">
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+
+  <p class="dummy-paragraph">Invocation of the component would look something like this:</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock @language="markup" @code="<Hds::DismissButton @onClick=\{{this.noop}} />" />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::DismissButton @onClick={{this.noop}} />
```

Comment:
as discussed, this probably should be something like: `<Hds::DismissButton {{on "click" this.onClickDismissButton}} />`

Reply:
Updated as such